scripting/include/srccoop/utils.inc

```diff
@@ -1070,3 +1070,8 @@ stock void SetBitOnInt(int& i, const int iBitIndex, const bool bSetBit)
 		i &= ~(1 << iBitIndex);
 	}
 }
+
+stock CBaseEntity GetEntityFromEHandlePointer(const Address pAddress)
```

Comment:
Would `LoadEntityFromHandleAddress` from sm1.12 work? We can add a static constructor to CBaseEntity.

Reply:
Fixed! Instead of adding a static constructor, I just went with a direct function call as we would be typically grabbing a EHandle from inside of a class.